openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -144,13 +144,13 @@ <h2 class="staff-info__title ui-staff__subtitle">
                     </tbody>
                 </table>
             {% elif workflow_status == "waiting" %}
-                <div class="staff-info__final__grade__score">{% trans "The submission is waiting for assessments." %}</div>
+                <div class="staff-info__final__grade__score"><p>{% trans "The submission is waiting for assessments." %}</p></div>
```

Comment:
Nit: I'd put the paragraph on its own line for all of these changes. These lines are already long, and paragraphs are semantically their own thing. It also makes the ``<p>`` tag more visible, as it is somewhat hidden when tucked away here.